### PR TITLE
Add FR-NL-1-03 to the NL validation plugin

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Reason for change
Added FR-NL-1.03 validation to NL validation plugin

#### Description of change
FR-NL-1.03: A DOCTYPE declaration MUST NOT be used in the filing instance document.

#### Steps to Test
CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: tests/resources/conformance_suites/nl_nt16/index.xml
Comment (aaroncameron-wk) [inline tests/resources/conformance_suites/nl_nt16/index.xml:16]: Whoops, I should have caught this on the last PR during QA.

Comment (derekgengenbacher-wf) [reply tests/resources/conformance_suites/nl_nt16/index.xml:16]: I caught it right after you merged it.

**Review by allenbross-wf — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

**Pull request merged**
